### ISSUE TITLE
Consider using dirs-dev

### ISSUE THREAD

**Issue by ckipp01** [lunatech-labs/course-management-tools#209]
[dirs-dev](https://github.com/dirs-dev/directories-jvm) is a great _tiny_ little Java library that takes care of making sure that configuration and cache directories follow the standards set by various OS's about where these things are supposed to go. I noticed that when using `cmtc` for the first time multiple new directories were created in my home directory. As an alternative, and arguably best practice, you could use something like the following:

```scala
val projectDirs = ProjectDirectories.from("", "lunatech", "cmt")

val defaultCacheDir = projectDirs.cacheDir // where you could put Courses
val defaultConfigDir = projectDirs.configDir // where you could put the default config
```
Again, this would ensure that the conventions for each OS are fully handled.

**Comment by eloots** [comment]
Great suggestion! We should do that asap...

**Comment by eloots** [comment]
Implemented via #218
